app/server.go

```diff
@@ -1380,9 +1400,16 @@ func (s *Server) doLicenseExpirationCheck() {
 			continue
 		}
 
+		T := i18n.GetUserTranslations(user.Locale)
+		ctaText := T("api.templates.license_up_for_renewal_renew_now")
+		if !status.IsExpandable {
```

Comment:
IsExpandable looks to rely on [the subscription status being active](https://github.com/mattermost/customer-web-server/blob/master/model/subscription.go#L255) which will never actually be the case after the license has expired (we cancel them at period end), so i don't think this path will ever fire